### ISSUE TITLE
Adding a linter?

### ISSUE THREAD

**Issue by erdeszt** [NikolajLund/scala-3-service-blueprint#4]
Wartremover supports scala 3 and it would've caught the issue of comparing unrelated types that I mentioned in #3

**Comment by NikolajLund** [comment]
Yup, I've been wanting to add Wartremover. Will take a stab at it
